token/core/fabtoken/wallet.go

```diff
@@ -346,12 +177,12 @@ func (w *ownerWallet) Remote() bool {
 }
 
 type issuerWallet struct {
-	tokenService *WalletService
+	tokenService *common.WalletService
```

Comment:
Passing a whole service into the wallet adds a dependency that would maybe better be avoided.
I checked why we need to pass the wallet service, and if I saw correctly, it's only because of `nogh.Service.Issue()` that calls `w.GetSigner()`. If this is the case, we could pass the walletService into `nogh.Service` directly and avoid passing the walletService altogether.
But it might be that it's more complex than that.